js/dropdowns.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
-/* global require, module, document */
-
 var $ = require('jquery');
-var perfectScrollbar = require('perfect-scrollbar/jquery')($);
+require('perfect-scrollbar/jquery')($);
```

Comment:
Just to make sure I'm understanding this correctly, this is providing the perfect-scrollbar library to the jQuery object itself ($) by using the `require` statement in this fashion?